sdk/appconfiguration/spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java

```diff
@@ -40,15 +40,15 @@
     private static final String PROPERTY_SOURCE_NAME = "azure-config-store";
 
     private static final String PATH_SPLITTER = "/";
-    private static Boolean startup = true;
+    private Boolean startup = true;
```

Comment:
Is there a reason this static has been removed? This should be static.

Reply:
Thank you for your careful check.

The purpose is to fix build error: not modify static field in instance method.
I found another way to fix the build error, please check the latest code.